CMSgov/mdct-mcr
# add count to modal drawer sections

## Description

If there are cards to display for modal drawers, the count should display above the cards (same view as regular program). If there are no cards, the header should still appear as normal but instead of a count it should read: '0 -- No {entity} entered' in the error-darker color.

### How to test
Add some data, see that count shows up correctly, when there's no data, see that '0 -- No {entity} entered' shows.

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
